packages/frontend/src/components/wallet/LinkDropSuccessModal.js

```diff
@@ -33,11 +33,11 @@ const Container = styled.div`
 
 `;
 
-const LinkDropSuccessModal = ({ open, onClose, linkdropAmount }) => {
+const LinkDropSuccessModal = ({ onClose, linkdropAmount }) => {
     return (
         <Modal
             id='near-drop-success-modal'
-            isOpen={open}
+            isOpen={linkdropAmount}
```

Comment:
'0' will be considered `true` if we refer to `isOpen` using truesy/falsy checks -- which I don't think we want?
Would prefer is `isOpen` is converted to a boolean right here, so that the `Modal` component gets `true` or `false` and _this_ component is responsible for making sure that whatever value is in `linkdropAmount` is coerced properly taking into account it's a string (not a Number) where we are passing it here.